base/applications/mspaint/lang/bg-BG.rc

```diff
@@ -191,6 +191,17 @@ BEGIN
     PUSHBUTTON "Отказ", IDCANCEL, 170, 24, 48, 14
 END
 
+IDD_FONTS DIALOG 0, 0, 225, 25
+CAPTION "Fonts"
+STYLE DS_CENTER | DS_MODALFRAME | WS_POPUPWINDOW | WS_CAPTION
+EXSTYLE WS_EX_TOOLWINDOW
+FONT 9, "MS Shell Dlg"
```

Comment:
By default, the font size is "8", except for CJK languages.

Reply:
Fixed.